src/index.js

```diff
@@ -39,95 +26,31 @@ nconf.argv()
     WEBSERVER_ENABLED: 'true'
   });
 
-function saveConfig(key, value) {
-  log.debug(`Saving config key: ${key}, value: ${value}`);
-  nconf.set(key, value);
-  nconf.save(err => {
-    fs.readFile(configFile, (err, data) => {
-      data = data || {};
-      console.dir(JSON.parse(data.toString()));
-    })
-  });
-}
+var adminUtils = require('./adminUtils.js');
```

Comment:
Probably move all the requires to the top of the file (same with DiscordBot, queues, and files below).